etc/ReplayOfflineConfiguration.py

```diff
@@ -260,7 +293,6 @@
 
 addExpressConfig(tier0Config, "HLTMonitor",
                  scenario=ppScenario,
```

Comment:
Just for my curiosity:  `diskNode="T2_CH_CERN",` has been omitted several time, what's the reason for that?

Reply:
So that the output will not be saved to disk I think.